core-services/prow/02_config/openshift/velero/_prowconfig.yaml

```
@@ -13,6 +13,9 @@ branch-protection:
             oadp-1.1:
               allow_force_pushes: true
               protect: true
+            oadp-1.2:
+              allow_force_pushes: true
+              protect: true
 tide:
   queries:
```

```suggestion
tide:
  merge_method:
    openshift/velero: squash
  queries:
```
Similar to
https://github.com/openshift/release/pull/38010/files#diff-3662ab9ef29d6d8e8c3e3e2bbe09019fad92c59c3e5c3eb488f0984f917d1060R11-R12